docs/user/dql/functions.rst

```diff
@@ -556,6 +583,8 @@ Argument type: INTEGER/LONG/FLOAT/DOUBLE
 
 Return type: Wider type between types of n and m if m is nonzero value. If m equals to 0, then returns NULL.
 
+Synonyms: %, MODULUS
+
```

Comment:
Update argument type list

Reply:
Could you please elaborate on this comment?

Reply:
Fixed